sensor_msgs/CompressedImage left
sensor_msgs/CompressedImage right